phpBB/adm/style/permissions.js

```diff
@@ -294,7 +323,8 @@ function set_role_settings(role_id, target_id) {
 	// Mark all options to no (unset) first...
 	mark_options(target_id, 'u');
 
-	for (var r in settings) {
+	for (const r in settings) {
```

Comment:
In all other `for ()`'s you use `let` instead of `const`.
Not sure if this one is intentional, just pointing it out.